test/service/test_manifest.py

```diff
@@ -1324,7 +1324,7 @@ def test_fetch_manifest(self, get_cached_manifest):
                 }
                 if format_ == ManifestFormat.curl:
                     expected_json['CommandLine'] = {
-                        'cmd.exe': None,
+                        'cmd.exe': f'curl.exe {generator._process_url(manifest_url)} | curl.exe --config -',
```

Comment:
```suggestion
                        'cmd.exe': f'curl.exe "{manifest_url}" | curl.exe --config -',
```
The `manifest_url` in this test is known and doesn't contain any characters that will be escaped so it can be specified without processing through `_process_url`. A test that verifies f(x) == f(x) doesn't help as much as a test that verifies f(x) == 'foobar'.
Consider changing `manifest_url` to a value that will be modified by both escape methods (`_process_url` & `shlex.quote`), and hardcoding the expected result here.

Reply:
>Consider changing manifest_url to a value that will be modified by both escape methods (_process_url & shlex.quote), and hardcoding the expected result here.

I am not sure what this would entail since the `_cmd_exe_quote` method only makes some assertions and surrounds the URL with double quotes.

Reply:
Point taken. My comment was made when I thought `_cmd_exe_quote` would be escaping the value in some manor.